docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-adding-adhoc-tasks-proc.adoc

```diff
@@ -1,2 +1,19 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-adding-adhoc-tasks-proc-{context}']
+= Adding ad hoc tasks 
+
+Ad hoc tasks are optional user or service tasks that are configured in a case definition. Because they are ad hoc, they must be triggered in some way, usually by a signal event or Java API call.
```

Comment:
ad hoc tasks can be pretty much any task and not only user or service. this restriction is for dynamic tasks

Reply:
ack. removed the user or service bit.